src/components/v5/frame/ColonyHome/partials/ReputationChart/utils.ts

```diff
@@ -107,16 +120,29 @@ export const getTeamReputationChartData = (
 export const getContributorReputationChartData = (
   contributorsList: ContributorItem[],
 ): ReputationChartDataItem[] => {
-  const topContributors = contributorsList
+  let topContributors = contributorsList
     .slice(0, WIDGET_TEAM_LIMIT)
     .map(({ walletAddress, user, reputation }, index) => {
       return {
         id: walletAddress,
         label: user?.profile?.displayName || '',
-        value: Number(reputation),
+        value: reputation || 0,
         color: getTeamHexColor(CONTRIBUTORS_COLORS_LIST[index]),
       };
-    });
+    })
+    .filter(({ value }) => value > 0);
+
+  if (topContributors.length <= WIDGET_TEAM_LIMIT) {
+    const adjustedValues = adjustPercentagesTo100(
+      topContributors.map((contributor) => contributor.value),
+      // getNormalisedDomainReputationPercentage rounds to two decimals
+      2,
+    );
+    topContributors = topContributors.map((contributor, idx) => ({
+      ...contributor,
+      value: adjustedValues[idx],
+    }));
+  }
 
   const reputationInOtherContributors = contributorsList
```

Comment:
I know it wasn't done by you, but now that I have noticed, maybe we can rename the current value in the `reduce` function from `team` to `contributor`

Reply:
Will do, I did it in other places but must have missed this one :)